cheatsheets/All-XSS-Payloads-Cheat-Sheet.md

```diff
@@ -0,0 +1,1378 @@
+ULTIMATE CROSS SITE SCRIPTING CHEAT SHEET
+
+Note: This is a technical sheet for research about directory- and path traversal attacks.
+Please continue the ultimate directory traversal cheat sheet list or contribute to update.
+This cheat sheet list goes out to assist pentesters, developers, researchers & whitehats.
+
+Tags to Trigger XSS Attacks:
```

Comment:
These lines should be commented out so they can easily be ignored when using this wordlist. Preferably, use `# `. Same goes for the other lines which are just comments.

Reply:
Updated the file with comments as #.